### PR TITLE
review

### PR DESCRIPTION
**Pull request by gaks1**
1. To do list repo pull request for partner review

### CODE REVIEW

File: src/modules/edittask.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    editInput.focus();
+
+    editInput.addEventListener('keypress', (e) => {
+      if (e.key === 'Enter') {
+        task.description = editInput.value;
+        storeTask();
+        const newLabel = document.createElement('label');
+        newLabel.setAttribute('for', task.index);
+        newLabel.textContent = task.description;
+        editInput.parentNode.replaceChild(newLabel, editInput);
```
Comment (Newtayo) [inline src/modules/edittask.js:20]: Kindly make this a function so you can call it anytime you need to run the edit function. Repeating a code does make one codebase DRY.

**Review by Newtayo — CHANGES_REQUESTED** [review]
## Highlights

- [ ] Good implementation of the Todo List Task. Adding, removing, and editing tasks works very well on the app.
- [ ] Cleared completed button also works well.
- [ ] Changes in task status are tracked in real time.
## Require Changes 
Kindly consider creating a function to implement the editing of tasks on the TODO app. Repeating of code does not make one codebase clean

File: src/modules/edittask.js (hunk 1 above)
Comment (gaks1) [reply src/modules/edittask.js]: Thank you

File: src/modules/edittask.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      }
+    });
+    editInput.addEventListener('blur', () => {
+      task.description = editInput.value;
+      storeTask();
+      const newLabel = document.createElement('label');
+      newLabel.setAttribute('for', task.index);
+      newLabel.textContent = task.description;
+      editInput.parentNode.replaceChild(newLabel, editInput);
```
[the remark this answers is not in the document]
Comment (gaks1) [reply src/modules/edittask.js]: Thank you

**Review by Newtayo — APPROVED** [review]
## Good Job #:100: 

There is nothing to add other than for you to merge your project

**Pull request merged**